cf_cli_java_plugin.go

```diff
@@ -168,10 +177,23 @@ func (c *JavaPlugin) execute(commandExecutor cmd.CommandExecutor, uuidGenerator
 	}
 
 	var remoteCommandTokens = []string{JavaDetectionCommand}
-
+	remoteFileFullPath := ""
+	localFileFullPath:= ""
```

Comment:
It looks like the code doesn't wasn't formatted with a code formatter. Can you please apply [`gofmt`](https://pkg.go.dev/cmd/gofmt)?